src/Variables.ts

```diff
@@ -449,7 +449,7 @@ export const Globals: GlobalVariables = {
   autoTalismanTimer: 0,
 
   autoChallengeTimerIncrement: 0,
-  corruptionTrigger: 1,
+  corruptionTrigger: 'illiteracy',
```

Comment:
possible to use an enum here?

Reply:
This feeds into `corruptionTrigger` which uses `keyof Corruptions` as argument as of this update